BraveShared/CertificatePinning.swift

```diff
@@ -0,0 +1,100 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+import Shared
+
+private let log = Logger.browserLogger
+
+public class PinningCertificateEvaluator: NSObject, URLSessionDelegate {
```

Comment:
Move this to `Client/Networking` folder

Reply:
If I move it, how do I access it from BraveShared? That's where the URPService is located.

Reply:
you are right, sorry